packages/ntcore-ts-client/src/lib/pubsub/prefix-topic.ts

```diff
@@ -0,0 +1,93 @@
+import { Util } from '../util/util';
+
+import { NetworkTablesBaseTopic } from './base-topic';
+
+import type { CallbackFn } from './base-topic';
+import type { PubSubClient } from './pubsub';
+import type {
+  AnnounceMessageParams,
+  NetworkTablesTypes,
+  SubscribeMessageParams,
+  SubscribeOptions,
+} from '../types/types';
+
+export class NetworkTablesPrefixTopic extends NetworkTablesBaseTopic<NetworkTablesTypes> {
+  readonly type = 'prefix';
+
+  /**
+   * Creates a new topic. This should only be done after the
+   * base NTCore client has been initialized.
+   * @param client - The client that owns the topic.
+   * @param name - The name of the topic.
+   */
+  constructor(client: PubSubClient, name: string) {
+    super(client, name);
+
+    const existingTopic = this.client.getPrefixTopicFromName(name);
+    if (existingTopic) {
+      return existingTopic;
+    }
+
+    this.client.registerTopic(this);
+  }
+
+  /** */
+  /* SUBSCRIBING */
+  /** */
+
+  /**
+   * Creates a new subscriber. This should only be called by the PubSubClient.
```

Comment:
Can you remove ` This should only be called by the PubSubClient.` from the subscribe functions in PrefixTopic and Topic?